docs/multi-cluster-ingress.md

```diff
@@ -27,55 +27,45 @@ We use [multi-cluster-ingress-nginx](https://github.com/karmada-io/multi-cluster
 
 ```shell
 # for HTTPS
-git clone https://github.com/karmada-io/multi-cluster-ingress-nginx.git
+$ git clone https://github.com/karmada-io/multi-cluster-ingress-nginx.git
 # for SSH
-git clone git@github.com:karmada-io/multi-cluster-ingress-nginx.git
+$ git clone git@github.com:karmada-io/multi-cluster-ingress-nginx.git
 ```
 
 #### Build and deploy ingress-nginx
 
 Using the existing `karmada-host` kind cluster to build and deploy the ingress controller.
 
 ```shell
-export KIND_CLUSTER_NAME=karmada-host
-kubectl config use-context karmada-host
-make dev-env
+# set the default cluster profile
+$ export KUBECONFIG=~/.kube/karmada.config
+# set the kind cluster name for ingress deploy
+$ export KIND_CLUSTER_NAME=karmada-host
+# the default context is switched to karmada-host
+$ kubectl config use-context karmada-host
+Switched to context "karmada-host".
+$ cd multi-cluster-ingress-nginx
+$ make dev-env
 ```
 
 #### Apply kubeconfig secret
 
-Create a Secret that contains the `karmada-apiserver` authentication credential in the following format:
-
-```yaml
-# karmada-kubeconfig-secret.yaml
-apiVersion: v1
-data:
-  kubeconfig: {data} # Base64-encoded
-kind: Secret
-metadata:
-  name: kubeconfig
-  namespace: ingress-nginx
-type: Opaque
-```
-
-You can get the authentication credential from the `/root/.kube/karmada.config` file, or use command:
-
-```shell
-kubectl -n karmada-system get secret kubeconfig -oyaml | grep kubeconfig | sed -n '1p' | awk '{print $2}'
-```
-
-Then apply yaml:
+Create a secret that contains the `karmada-apiserver` authentication credential:
 
 ```shell
-kubectl apply -f karmada-kubeconfig-secret.yaml
+# get the 'karmada-apiserver' kubeconfig information and direct it to file /tmp/kubeconfig
+$ kubectl -n karmada-system get secret kubeconfig --template={{.data.kubeconfig}} | base64 -d > /tmp/kubeconfig.yaml
+# create secret with name 'kubeconfig' from file /tmp/kubeconfig
+$ kubectl -n ingress-nginx create secret generic kubeconfig --from-file=kubeconfig=/tmp/kubeconfig
+secret/kubeconfig created
```

Comment:
Have you tested with it and tried to run the entire MCI feature?

Reply:
This step has been tested. The original step was to obtain the cluster certificate from `.kube/karmada.config`. However, I think we should refer to kubeconfig secret in the karmada-system namespace directly here, because config may be external access. However, when MCI is in the karmada-host cluster, it can go directly to service discovery